content/en/docs/chart_best_practices/values.md

```diff
@@ -162,3 +162,106 @@ serverPort: 9191
 Beginning each comment with the name of the parameter it documents makes it easy
 to grep out documentation, and will enable documentation tools to reliably
 correlate doc strings with the parameters they describe.
+
+### Nested Properties:
+
+When documenting nested properties, use the "fully qualified" name of the property
```

Comment:
"fully qualified" isn't great but I couldn't think of a better descriptor at the time. Maybe
```suggestion
When documenting nested properties, use the complete path of the property
```
?